compiler/src/iree/compiler/Codegen/TransformDialectStrategies/GPU/Common.cpp

```diff
@@ -69,7 +69,7 @@ using iree_compiler::gpu::StagedReductionStrategy;
 int64_t mlir::iree_compiler::gpu::scaleUpByBitWidth(int64_t value,
                                                     int64_t bitWidth) {
   assert((bitWidth & bitWidth - 1) == 0 && "bitWidth must be a power of 2");
-  return std::max((value * 32) / bitWidth, 1l);
+  return std::max((value * 32) / bitWidth, 1ll);
```

Comment:
Lei fixed a few of these differently: https://github.com/iree-org/iree/pull/11724, https://github.com/iree-org/iree/pull/11739

```diff
-                 ? std::make_pair(0l, vectorSize)
+                 ? std::make_pair(int64_t(0), vectorSize)
```

Beyond having more CI builds and compiler warnings, this stuff could be caught during code review too.

Reply:
(@antiagainst )

Reply:
Oh, and this seems to be failing to compile... might need the cast like what Lei did?

https://github.com/iree-org/iree/actions/runs/3876917220/jobs/6611330456
```
/work/compiler/src/iree/compiler/Codegen/TransformDialectStrategies/GPU/Common.cpp:72:10: error: no matching function for call to 'max'
  return std::max((value * 32) / bitWidth, 1ll);
         ^~~~~~~~
/usr/bin/../lib/gcc/x86_64-linux-gnu/9/../../../../include/c++/9/bits/algorithmfwd.h:370:5: note: candidate template ignored: deduced conflicting types for parameter '_Tp' ('long' vs. 'long long')
    max(const _Tp&, const _Tp&);
    ^
```